.gitignore

```
@@ -39,10 +39,16 @@ ENV/
 env.bak/
 venv.bak/
 env.d/development/crowdin
+env.d/terraform
```

```suggestion
env.d/terraform
!env.d/terraform/example.dist
```

You should not ignore `example.dist`

---

I have remove example.dist as this file is standard in openstack and can be exported from keystone.

---

So I think you should update the comment (L:121) within `_config.sh` when there is no existing terraform project.
```
        echo "To create one, copy and customize \"env.d/terraform/example.dist\"" \
```

---

good catch :pray:
done